lib/views/create-bug.js

```diff
@@ -80,38 +79,33 @@ function drawAttachments() {
 // API needs anyway, may want to avoid encoding until they are sent
 function inputChanged(e) {
   var files = e.target.files;
+  var queue = [];
```

Comment:
nit: this is a bad name as it invokes the idea of the queue data structure, which it doesn't seem like it really is. Maybe just call this ```attachments``` instead?